internal/filter/on_collab_style.go

```diff
@@ -0,0 +1,17 @@
+package filter
+
+import (
+	"github.com/clean-code-projects/co-coders-api/internal/criteria"
+	"github.com/clean-code-projects/co-coders-api/internal/user"
```

Comment:
Why do some imports get this fully qualified name and other imports just a simple name?

Reply:
The simple names are for standard library imports